src/pages/CommunicatingWithGraphQLAPI.js

```diff
@@ -10,36 +10,40 @@ const content = markdown(markdownOptions)`
 
 In this section, we’ll explain the core concepts you need to know when working with a GraphQL API. 
 
+
+<!--
```

Comment:
This TODO block causes a syntax error. Let's either update or remove it?

Reply:
I removed it for now, we're planning to update graphql-up soon, so then this section can be updated again :)